File: missingLetter.js
const getMissingLetter = (words) => {
  let prevCode = undefined;
  let result = [];
  for (let i = 0; i < words.length; i++) {
    if (!prevCode) {
      prevCode = words.charCodeAt(i);
      continue;
    }

    if (words.charCodeAt(i) - prevCode !== 1) {
      const missingCode = words.charCodeAt(i) - 1;
      const missingLetter = String.fromCharCode(missingCode);
      result.push(missingLetter);
      prevCode = missingCode + 1;
    } else {
      prevCode = words.charCodeAt(i);
    }
  }
  if (result.length) {
    return result.join(",");
  }
  return undefined;
};

console.log(getMissingLetter("bcefghijklmnopqrsuvxyz"));

File: python/missingLetter.py
def missingLetter(words):
    prevCode = None
    result = []
    for i in range(len(words)):
        if prevCode == None:
            prevCode = ord(words[i])
            continue
        if (ord(words[i]) - prevCode) != 1:
            missingCode = ord(words[i]) - 1
            missingLetter = chr(missingCode)
            result.append(missingLetter)
            prevCode = missingCode + 1
        else:
            prevCode = ord(words[i])

    if len(result):
        return ','.join(result)
    return None

print(missingLetter('bdefghijklmnpqrstuvxyz'))